src/controllers/ExtensionController.ts

```diff
@@ -322,6 +322,8 @@ export class ExtensionController implements IDisposable {
       this._outputChannel,
       this._toaster
     );
+
+    this._context.subscriptions.push(this._pipServerController);
```

Comment:
This needed to be added to be consistent with other disposables, but it doesn't fix the orphaned terminal problem at all. Attempting to close the terminals as the extension is deactivating seems to have no effect.